sources/packages/backend/apps/db-migrations/src/sql/NotificationMessages/Insert-offering-change-request-inprogress-with-student-message.sql

```diff
@@ -0,0 +1,8 @@
+INSERT INTO
+  sims.notification_messages(id, description, template_id)
+VALUES
+  (
+    13,
+    'Offering Change Request Inprogress With Student',
```

Comment:
Please fix the "Inprogress".
Was this message reviewed by the business? The idea is that these messages can potentially be shown to the user some day.

Reply:
I haven't reviewed with the business yet. I will check it with @JasonCTang